docs/docs/env-vars.md

```diff
@@ -10,6 +10,11 @@ route: "/env-vars"
 **Description**: Path to the `kubectl` tool executable  
 **Default**: `$PATH/kubectl`
 
+### TANKA_JB_PATH
+
+**Description**: Path to the `jb` tool executable
+**Default**: `$PATH/jb`
+
```

Comment:
Would you mind putting this below / above the other `kubectl` entry so they are grouped by application they refer to? :hugs:

Reply:
Cool; will do it in a sec ;-)